integrations/ingress-nginx/documentation.yaml

```diff
@@ -0,0 +1,90 @@
+exporter_type: included
+app_name_short: Ingress NGINX
+app_name: {{app_name_short}}
+app_site_name: Ingress NGINX
+app_site_url: https://kubernetes.github.io/ingress-nginx
+exporter_name: the Ingress NGINX exporter
```

Comment:
should this maybe be "controller"? That seems to be how it's referred to.

Reply:
if so, it might make the doc flow more naturally for people familiar with this if some of the occurrences of {{app_name_short}} in prose were replaced with {{exporter_name}}.